docs/source/Setup/PyPy.md

```diff
@@ -0,0 +1,24 @@
+# PyPy Support
+
+## Notes about PyPy support
+
+Although not actively supported PyPy should work without problems, or at least not too many. Every 
+night the official test suite is run against PyPy so if you want to use it you can start checking
+the latest [CI results against master branch](https://github.com/evennia/evennia/actions/workflows/github_action_pypy_test_suite.yml).
+
+## Additional requirements
+
+Based on your setup, you could need additional packages to be installed along the normal
+requirements.
```

Comment:
This seems pretty inadequate. I'd expect to at least see full install instructions for getting Evennia running with pypy on a debian-based Linux distro, for example.

Reply:
ok, I can do that. The reason I didn't wrote so much there is existing install instructions are perfectly fine for pypy too

Reply:
ok, I've missed the part where docs actually explain how to install python, and what. Professional deformation I guess :D I'll do that